src/test/streams_tests.cpp

```diff
@@ -9,8 +9,7 @@
 #include <boost/assign/std/vector.hpp> // for 'operator+=()'
 #include <boost/assert.hpp>
 #include <boost/test/unit_test.hpp>
-                    
-using namespace std;
+
 using namespace boost::assign; // bring 'operator+=()' into scope
```

Comment:
IIRC in C++11, you can initialise this vector with

```C++
key = {'\x00','\x00'};
```

But this is probably outside of the scope of this PR.